### PR TITLE
Changed defaults, removed UseSSL

### PR DESCRIPTION
**Pull request by dysk**
* Defaults changed when creating/updating a queue: CustomLayout set to dark theme, MaxNoOfRedirectsPrQueueId to 3, QueueNumberValidityInMinutes to 30.

* UseSSL parameter is no longer sent since the use of https will be enforced.

### CODE REVIEW

**Comment by dysk** [comment]
@kaspergrubbe please review and merge if it's OK

File: lib/queue_it/api/event.rb [hunk 1]
```diff
@@ -107,17 +107,16 @@ def queue_attributes(pre_queue_start_time:, start_time:, end_time:, know_user_se
...
           "AfterEventLogic"              => "RedirectUsersToTargetPage",
           "AfterEventRedirectPage"       => "",
-          "UseSSL"                       => "Auto",
           "JavaScriptSupportEnabled"     => "False",
           "TargetUrlSupportEnabled"      => "True",
           "SafetyNetMode"                => "Disabled",
           "KnowUserSecurity"             => "MD5Hash",
           "KnowUserSecretKey"            => know_user_secret_key,
-          "CustomLayout"                 => "Default layout by Queue-it",
+          "CustomLayout"                 => "Dark theme",
```
Comment (kaspergrubbe) [inline lib/queue_it/api/event.rb:119]: Are we sure this is a theme that Queue-it provides?

File: lib/queue_it/api/event.rb [hunk 2]
```diff
@@ -107,17 +107,16 @@ def queue_attributes(pre_queue_start_time:, start_time:, end_time:, know_user_se
           "EventStartTime"               => format_time( utc_start_time(start_time) ),
           "EventEndTime"                 => format_time( utc_end_time(start_time, end_time) ),
           "EventCulture"                 => event_culture_name,
-          "MaxNoOfRedirectsPrQueueId"    => "1",
+          "MaxNoOfRedirectsPrQueueId"    => "3",
           "QueueNumberValidityInMinutes" => "#{queue_number_validity_in_minutes}",
```
Comment (kaspergrubbe) [inline lib/queue_it/api/event.rb:111]: Consumers of this code could just pass this from when they use it.

Comment (kaspergrubbe) [reply lib/queue_it/api/event.rb:111]: But 15 or 30 minutes doesn't really change anything for me, so it's fine.

File: spec/queue_it/api/event_spec.rb [hunk 3]
```diff
@@ -142,17 +142,16 @@ module Api
...
             "AfterEventLogic"              => "RedirectUsersToTargetPage",
             "AfterEventRedirectPage"       => "",
-            "UseSSL"                       => "Auto",
             "JavaScriptSupportEnabled"     => "False",
             "TargetUrlSupportEnabled"      => "True",
             "SafetyNetMode"                => "Disabled",
             "KnowUserSecurity"             => "MD5Hash",
             "KnowUserSecretKey"            => know_user_secret_key,
-            "CustomLayout"                 => "Default layout by Queue-it",
+            "CustomLayout"                 => "Dark theme",
```
Comment (kaspergrubbe) [inline spec/queue_it/api/event_spec.rb:154]: Is this a default theme by Queue-it? If yes, it does seem a bit daft that it isn't named `Dark theme by Queue-it`.

Also, should this be configurable? It seems like we're changing behavior.

**Pull request merged**